public/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+    font-family: Arial;
+    margin-left: 5%;
+    margin-top: 20%;
+  }
+  
+  * {
+    box-sizing: border-box;
+  }
+  
+  form.search input[type=text] {
+    padding: 10px;
+    font-size: 17px;
+    border: 1px solid grey;
+    float: left;
+    width: 80%;
+    background: #f1f1f1;
+  }
+  
+  form.search button {
+    float: left;
+    width: 20%;
+    padding: 10px;
+    background: #2196F3;
+    color: white;
+    font-size: 17px;
+    border: 1px solid grey;
+    border-left: none;
+    cursor: pointer;
+  }
+  
+  form.search button:hover {
+    background: #0b7dda;
+  }
+  
+  form.search::after {
+    content: "";
+    clear: both;
+    display: table;
+  }
+
+  .search{
+
```

Comment:
Please delete all this free space above and below these two lines.

Reply:
@jema28  fixed thanks